lib/models/ReplicationConfiguration.ts

```diff
@@ -155,7 +155,7 @@ export default class ReplicationConfiguration {
     _buildRuleObject(rule: XMLRule) {
         const base = {
             id: '',
-            prefix: rule.Prefix[0],
+            prefix: rule.Prefix?.[0] ?? '',
```

Comment:
this is duplicated with `_parsePrefix`, can we avoid this and do the "parsing" just once?

(though `parsePrefix` name is actually incorrect, it really just validates the prefix...)

Reply:
The caller of _buildRuleObject is already doing all the parsing verifications. I think it would be complicated to use _parsePrefix in this function, as it would now have to return an error.

What I would do is rename the rule parameter like this : 

```suggestion
    _buildRuleObject(rule: XMLRule) {
    _buildRuleObject(validRule: XMLRule) {
```